railties/lib/rails/generators/rails/app/templates/config/puma.rb.tt

```diff
@@ -14,7 +14,7 @@ threads min_threads_count, max_threads_count
 # Specifies that the worker count should equal the number of processors in production.
 if ENV["RAILS_ENV"] == "production"
   worker_count = ENV.fetch("WEB_CONCURRENCY") { Concurrent.physical_processor_count }
-  workers worker_count if worker_count > 1
+  workers worker_count if worker_count && worker_count.to_i > 1
```

Comment:
```suggestion
  workers worker_count if Integer(worker_count) > 1
```

`WEB_CONCURRENCY=ten` should probably raise an error, rather than running one process.